README.md

```diff
@@ -143,7 +157,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory, change to the most recent release tag, and build:
 
         cd aeon
-        git checkout v0.12.2.1
+        git checkout v0.12.6.0-aeon
```

Comment:
Maybe could you please also update the number in the "Recommended Aeon version" column of the Aeon version table?

Reply:
Done!